src/components/MainPage.js

```diff
@@ -0,0 +1,188 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Button from "@mui/material/Button";
+import CameraIcon from "@mui/icons-material/PhotoCamera";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Labels from "./Labels";
+import { Chip } from "@mui/material";
+import Progress from "./Progress";
+import reactJsLogo from "../logoImages/react-logo.png";
+import jsLogo from "../logoImages/js.svg";
+import dataStructureLogo from "../logoImages/data-structure.png";
+
+const quizCards = [
+  {
+    id: "react-basic",
+    title: "React Basics",
+    category: "react",
+    level: "basic",
+    labels: ["frontend"],
+    image: reactJsLogo,
+  },
+  {
+    id: "react-middle",
+    title: "React Mid-Level",
+    category: "react",
+    level: "mid-level",
+    labels: ["frontend"],
+    image: reactJsLogo,
+  },
+  {
+    id: "js-basic",
+    title: "JS Basics",
+    category: "javaScript",
+    level: "basic",
+    labels: ["frontend", "backend"],
+    image: jsLogo,
+  },
+  {
+    id: "js-functions",
+    title: "JS Functions",
+    category: "javaScript",
+    level: "basic",
+    labels: ["frontend", "backend"],
+    image: jsLogo,
+  },
+  {
+    id: "js-middle",
+    title: "JS Mid-Level",
+    category: "javaScript",
+    level: "mid-level",
+    labels: ["frontend", "backend"],
+    image: jsLogo,
+  },
+  {
+    id: "data-structures",
+    title: "Data structures",
+    category: "data structures",
+    level: "middle",
+    labels: ["frontend", "backend"],
+    image: dataStructureLogo,
+  },
+];
+const userProgress = {
```

Comment:
I think `userProgress` can be confusing as it refers to progress of a single user. Maybe you could rename it so it's more clear?